File: product/src/app/model/orderItem.ts
export class OrderItem {

    //Table ID
    id: number = 0;

    //Customer Data
    orderId: number = 0;
    customerId: number = 0;
    customerName?: string = "";

    //Product Data
    productId: number = 0;
    productName: string = "";
    imageFile: string = "";
    quantity: number = 0;
    price: string = "0.0";
    uom: string = "";
  
    //SKU??

    //status
    status: string ="not_selected";

    //For UI
    showConfirm?: boolean; //For UI
}

File: ecommerce/src/main/java/com/matoza/model/OrderItem.java
package com.matoza.model;
import lombok.Data;

@Data
public class OrderItem {

    //Customer Data
    int id;
   // int orderId;
   int customerId;
    String customerName;
    int orderId;

    //Product Data
    int productId;
    String productName;
    String imageFile;
    double quantity;
    String uom;
    double price;
    String status;

}

